src/data/forecast.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
If we're using the API to get the data now this is redundant and can be deleted

Reply:
forecast.json deleted.